service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -72,6 +80,7 @@ public void execute(CommandMessage request) {
                 });
     }
 
+    @SuppressWarnings("PMD")
```

Comment:
PMD is complaining about the cognitive load and TBH this method SHOULD be refactored. I did not want to jump the gun without getting your opinion first but I can definitely do that. Let me know @gregmeldrum